src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/pear/PearDependencyFinder.groovy

```diff
@@ -96,15 +96,34 @@ class PearDependencyFinder {
     }
 
     private Set<DependencyNode> createPearDependencyNodeFromList(String list, List<String> dependencyNames) {
+        Set<DependencyNode> parsedDependencies = []
+        String[] channelList = list.split('=')
+        channelList -= ''
+
+        channelList.eachWithIndex { String channelItem, row ->
+            if (row != 0) {
+                parsedDependencies.addAll(parseChannelBlock(channelItem, dependencyNames))
+            }
+        }
+
+        parsedDependencies
+    }
+
+    private Set<DependencyNode> parseChannelBlock(String list, List<String> dependencyNames) {
         Set<DependencyNode> childrenNodes = []
         String[] dependencyList = list.split(System.lineSeparator())
```

Comment:
I think the final dependencyList could be better constructed from a method to accomplish the cleaning, splitting, etc. Also, this file has some .each idioms and here, for reasons unknown, you are using a for loop - I'd prefer at least internal consistency within the file. I think overall, we are favoring the .each idiom lately.